app/qml/map/SplittingTools.qml

```diff
@@ -0,0 +1,106 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Shapes 1.14
+
+import QgsQuick 0.1
+import lc 1.0
+
+import "../"
+
+/**
+  * RecordingTools is a set of tools that are used during recording/editing of a geometry.
```

Comment:
Shouldn't it be `SplittingTools`? The rest of the comment also from another tool.